source/patterns/@aws-solutions-constructs/aws-eventbridge-kinesisstreams/test/integ.eventbridge-kinesisstreams-existing-eventbridge.ts

```diff
@@ -0,0 +1,42 @@
+/**
```

Comment:
Shouldn't this be existing-eventbus, not existing-eventbridge?

Reply:
fixed them and updated the PR